public/lib/content-service.js

```diff
@@ -23,8 +24,25 @@ angular.module('wfContentService', ['wfHttpSessionService', 'wfVisibilityService
                         "picture": "Picture",
                         "audio": "Audio",
                         "atom": "Video/Atom"
-                    });
-                };
+                    }); 
+                }
+
+                provideStandardAndNewFormats(){             
+                    return Promise.resolve({
+                        "article": "Article",
+                        "keyTakeaways": "Key Takeaways",
+                        "liveblog": "Live blog",
+                        "gallery": "Gallery",
+                        "interactive": "Interactive",
+                        "picture": "Picture",
+                        "audio": "Audio",
+                        "atom": "Video/Atom"
+                    })} 
+
+                getTypes() {
+                    return wfPreferencesService.getPreference('featureSwitch').then((isSwitchActive) => {
```

Comment:
since you've abstracted the feature switch logic into its own service (nice one btw 👏 ) might be worth using that here to keep everything nicely encapsulated (albeit with a function exposed to retrieve the feature switch value)